### DISCUSSION TITLE
Running a paired set of parameters

### DISCUSSION THREAD

**Discussion by alechen28** [snl-dakota/dakota#59] [Q&A] [answered]
Hello,

If I wanted to run a paired set of parameters, how would I do that? For example, I would be altering a cylinder's length and diameter, but I want to hit certain masses. Right now, I think the code below would run all the combinations of the cylinder length and width, but what if I wanted to pair the variables together in the run, so 6,3 and 7,4 and 8,5 ? 

```
method
  multidim_parameter_study
    partitions = 2 2

variables
  discrete_design_set         
   real 2
	  descriptors 'cylinder_L' 'cylinder_dia'
	  elements_per_variable 3 3
	  elements 6 7 8
	           3 4 5
```

**Comment by jadamstephens** [comment, marked as answer]
Dakota does not provide a way to pair variables explicitly. If what you want is to run a list of cylinder lengths with corresponding cylinder diameters, you could perform a [list_parameter_study](https://snl-dakota.github.io/docs/6.18.0/users/usingdakota/reference/method-list_parameter_study.html):

```
method
  list_parameter_study
    list_of_points
      6  3
      7  4
      8  5

variables
  active_design 
  continuous_design 2
    descriptors 'cylinder_L'  'cylinder_dia'
```
